specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3258,6 +3258,76 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/riskyIp/blobUris": {
+      "get": {
+        "tags": [ "Reports" ],
+        "description": "Gets all Risky IP report URIs for the last 7 days.",
+        "x-ms-examples": {
+          "services_listAllRiskyIpDownloadReport": {
+            "$ref": "./examples/RiskyIpDownload.json"
+          }
+        },
+        "operationId": "services_listAllRiskyIpDownloadReport",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },          
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Risky IP report URIs for the last 7 days.",
+            "schema": {
+              "$ref": "#/definitions/RiskyIPBlobUris"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/riskyIp/blobUri": {

```

Comment:
doesnt look like Kris's comments have been addressed? What is "riskyIp"? Is it a resource type? or is it an instance of "reports" resource type. Feel free to ping me to close on this - alias gauravbh

Reply:
RiskyIP is a type of report.